### PR TITLE
added node logic

### PR DESCRIPTION
**Pull request by hackaugusto**
Entry point for the state changes.

All state changes will be dispatched through a single write-ahead-log instance, this is the top-level logic  used to handle all state changes.

### CODE REVIEW

File: raiden/transfer/node.py [hunk 1]
```diff
@@ -0,0 +1,664 @@
...
+
+    if payment_network_state:
+        token_network_state = payment_network_state.tokenaddresses_to_tokennetworks.get(
+            token_network_identifier
+        )
+
+    return payment_network_state, token_network_state
+
+
+def get_token_network(node_state, payment_network_identifier, token_network_identifier):
```
Comment (palango) [inline raiden/transfer/node.py]: How is this different to the `get_token_network` function in `transfer/views.py`?

Comment (hackaugusto) [reply raiden/transfer/node.py]: at the moment, not different at all

Comment (palango) [reply raiden/transfer/node.py]: Will that change in the future?

Comment (hackaugusto) [reply raiden/transfer/node.py]: views are a different thing: https://github.com/raiden-network/raiden/pull/1288#issuecomment-370521424

the data returned by the views must be treated as immutable.

Comment (hackaugusto) [reply raiden/transfer/node.py]: anything else?

**Pull request merged**